src/utils/UrlUtils.test.ts

```diff
@@ -34,6 +34,16 @@ describe("UrlUtils", () => {
             expect(resultObj).toHaveProperty("foo", "test");
         });
 
+        it("should return query params when hash routes are used", () => {
+            // act
+            const result = UrlUtils.readParams("http://app/#/path?code=foo&status=bar");
```

Comment:
This is not the correct way to define urls with query parameter and imo it is not our job to deal with this..

Everything after the `#` is part of the hash and that includes the `?` etc. Means in technical terms this URL has no query.

Only the application specific hash based router knows about this and thus it must do URL parsing before our fallback is run.

To fix the test, please put the ? and it's values before the # so we ensure that this works (removing the change from below).

Reply:
De PR can be deleted I see I can fix this in our angular Code by passing a modified version of the default window.location.href